README.rst

```diff
@@ -51,7 +54,12 @@ dependencies to do basic tests is ``sphinx`` and ``sphinx_rtd_theme``
 (debian packages: ``python-sphinx`` and ``python-sphinx-rtd-theme``).
 
 HTML output is in ``_build/html/index.html``, and other output formats
-are available as well.
+are available as well. 
+
+.. To view built HTML output:
+.. - Linux ``xdg-open _build/html/index.html`` 
+.. - macOSX ``open _build/html/index.html``
+.. - Windows ``start _build/html/index.html``
```

Comment:
I recently learned of sphinx-autobuild, which could be recommend while we are at it.